multipart/src/main.rs

```diff
@@ -1,75 +1,53 @@
-use std::cell::Cell;
-use std::fs;
-use std::io::Write;
+extern crate actix_web;
+extern crate actix_multipart;
+extern crate serde;
 
-use actix_multipart::{Field, Multipart, MultipartError};
-use actix_web::{error, middleware, web, App, Error, HttpResponse, HttpServer};
-use futures::future::{err, Either};
-use futures::{Future, Stream};
+#[macro_use(concat_string)]
+extern crate concat_string;
+
+use std::io::Write;
+use actix_multipart::Multipart;
+use actix_web::{middleware, web, App, Error, HttpResponse, HttpServer};
+use futures::{StreamExt};
+use std::cell::Cell;
+use serde::Deserialize;
 
 pub struct AppState {
     pub counter: Cell<usize>,
 }
```

Comment:
Based on https://github.com/actix/examples/pull/196#discussion_r353901959

Could you kill this struct off please? I don't think it's getting used in the example and it's misleading.